### PR TITLE
Add field group info in organization

### PR DESCRIPTION
**Pull request by ostcar**

### CODE REVIEW

**Comment by ostcar** [comment]
@FinnStutzenstein @tsiegleauq  Please merge this as fast as possible. I don't want to do this again if there are any merge conflicts

File: docs/datavalidator/check_json.py [hunk 1]
```diff
@@ -202,6 +202,9 @@ def is_structured_field(self, field: str) -> bool:
 
     def is_normal_field(self, field: str) -> bool:
         return "$" not in field
+    
+    def is_calculated_field(self, field: str) -> bool:
+        return field == "content"
```
Comment (ostcar) [inline docs/datavalidator/check_json.py:207]: See #6195

File: docs/models.yml [hunk 2]
```diff
@@ -1190,382 +1531,506 @@ list_of_speakers:
...
     type: relation
     to: motion/derived_motion_ids  # Note: The related motions may not be in the same meeting
+    restriction_mode: B
   derived_motion_ids:
     type: relation-list
     to: motion/origin_id  # Note: The related motions may not be in the same meeting
-  all_origin_ids: number[]  # Calculated: All parents (origin_id), grandparents, etc. of this motion.
-  all_derived_motion_ids: number[]  # Calculated: All children (derived_motion_ids), grandchildren, etc. of this motion.
+    restriction_mode: B
+  all_origin_ids:
```
Comment (FinnStutzenstein) [inline docs/models.yml:1739]: `restriction_mode` is missing here and below

File: docs/models.yml [hunk 3]
```diff
@@ -1769,256 +2292,351 @@ poll:
...
       field: poll_ids
     equal_fields: meeting_id
+    restriction_mode: A
   option_ids:
     type: relation-list
     to: option/poll_id
     on_delete: CASCADE
     equal_fields: meeting_id
+    restriction_mode: A
+
```
Comment (FinnStutzenstein) [inline docs/models.yml:2358]: please remove

**Review by FinnStutzenstein — CHANGES_REQUESTED** [review]
Two small remarks. Otherwise ok. I think we does not have to wait for the backend since the models-generation is a manual step.

**Comment by FinnStutzenstein** [comment]
I'll enhance the models-validator the next days. Maybe I'll find some issues then ;)

File: docs/models.yml [hunk 4]
```diff
@@ -2028,26 +2646,32 @@ mediafile:
...
     type: template
     fields:
       type: relation
       to: meeting/font_$_id
+    restriction_mode: B
 
 projector:
-  id: number
-  name: string
+  id:
```
Comment (ostcar) [inline docs/models.yml:2671]: I forgot the projector

File: docs/models.yml (hunk 2 above)
Comment (ostcar) [reply docs/models.yml:1739]: The two fields are not documented in the wiki. Which group should I use?

https://github.com/OpenSlides/OpenSlides/wiki/motion-restrictions

File: docs/models.yml (hunk 3 above)
Comment (ostcar) [reply docs/models.yml:2358]: done

File: docs/models.yml (hunk 2 above)
Comment (FinnStutzenstein) [reply docs/models.yml:1739]: Oh, the field `forwarding_tree_motion_ids` was removed and replaced with both of them; needs a fix in the wiki, too.

File: docs/models.yml (hunk 4 above)
Comment (ostcar) [reply docs/models.yml:2670]: done

File: docs/models.yml (hunk 2 above)
Comment (ostcar) [reply docs/models.yml:1739]: So maybe it would be a good idea if the fields where only defined in the models.yml 😉

done

**Review by FinnStutzenstein — APPROVED** [review]

**Pull request merged**